.circleci/config.yml

```diff
@@ -32,8 +32,8 @@ jobs:
           - ~/app/node_modules
       - run: yarn packages:styles:build
       - run: yarn packages:components:build
-      - run: yarn build
-      - run: xvfb-run -a yarn test
+      - run: yarn affected:build
+      - run: xvfb-run -a yarn affected:test
```

Comment:
switch case:

 - on master: `yarn affected:build --all`
 - on branches for pull requests: `yarn affected:build --base=origin/master --head=HEAD`